File: src/main/java/me/arrush/javabase/databases/PGDatabase.java
package me.arrush.javabase.databases;

import java.sql.SQLException;

@SuppressWarnings("WeakerAccess")
public final class PGDatabase extends Database {

    PGDatabase(int port, String dbName, String username, String pass) throws ClassNotFoundException, SQLException {
        super(dbName, DatabaseType.POSTGRESQL, "org.postgresql.Driver", Database.getPostgresConnection(port, dbName, username, pass));
    }
}

File: src/main/java/com/arrushc/javabase/entities/database/PGDatabase.kt
package com.arrushc.javabase.entities.database

import com.arrushc.javabase.entities.Database

class PGDatabase internal constructor(port: Int, dbName: String, username: String, pass: String, ssl: Boolean = false) : Database(dbName, DatabaseType.POSTGRESQL, "org.postgresql.Driver", getPostgresDataSource(port, dbName, username, pass, ssl))